packages/audio_mixer/src/lib.rs

```diff
@@ -132,7 +132,7 @@ impl<Src: Debug + Clone + Eq + Hash> AudioMixer<Src> {
         for (i, slot) in self.outputs.iter().enumerate() {
             if let Some(OutputSlotState { audio_level, source }) = slot {
                 if let Some((_, _, lowest_slot_audio_level)) = &mut lowest {
-                    if *audio_level < *lowest_slot_audio_level || (*audio_level == *lowest_slot_audio_level) {
```

Comment:
Seems to I missed some logic here but it not clearly then can you put TODO here? We need to process some case we have same audio_level. Just check with smaller only

Reply:
Added a TODO logic.